### DISCUSSION TITLE
The 2nd law and time reversibility

### DISCUSSION THREAD

**Discussion by josePereiro** [josePereiro/TankeFactory#16] [Discussions]
## Discussion

- Given the time reversibility of motion equations, the ensemble argument that the equilibrium ($\pm$ fluctuations) is exponentially more likely to be observed because it is supported by more microstates, breaks. 
- The time reversibility argument makes that for any trajectory increasing the entropy, there is one decreasing it.  
- So, why it is the system more likely to be in one that leads to equilibrium?
- The problem can be solved by separating the arguments about the trajectories from the arguments about the properties of the trajectories. 
- The 2nd law is a property of the trajectories, not of the ensemble of trajectories. 

> #Figure 1
> A sketch of a feasible trajectory for an isolated system. The system evolve from one microstate to the next following its equations of motion, each microstate is supporting a given entropy value ($S$). 
> <img width="500" alt="Screenshot 2024-11-22 at 7 31 22 PM" src="https://github.com/user-attachments/assets/d0d99f73-edbb-419a-8bc9-2c6ceb187936">

- The key point (see Figure 1) is that we claim that this is how a typical trajectory looks. Most trajectories expend a larger fraction of time in a high entropy region of the phase space. 
- Note that this is true independently of the direction of time. 
- The trajectories where this is not the case are unfeasible or exponentially unlikely as the system size increases. 
- That is, it is not the probability of the trajectory what determine the 2nd law. 
- It is the probability that you are in a region of a trajectory where the entropy is decreasing for a significant period time.  

## Questions

- Definition/Meaning of entropy outside the equilibrium?
- Similar Ideas in Literature?
- Possible 'simple' Simulations?

**Comment by ta-rekt** [comment]
For a meaning of entropy outside equilibrium, I think there is a consensus in the stochastic/information thermo community that at least for isothermal processes, Shannon entropy does the job.

When you say "high entropy region of phase space", this is subjective unless you specify an observable. Any partitions of phase space are induced by macroscopic observables, in a more or less rigorous way depending on who you ask (Parrondo tried to propose a rigorous way to do it).

You're trying to frame the problem in terms of properties of trajectories, instead of properties of phase space, right? Maybe the notion of "typicality" in information theory can help here. You can specify if a trajectory, or any piece of it, are typical, in a well defined sense. It's something like this (from wikipedia, look up "typical set") If you take a trajectory $X = (x_1, x_2, \dots, x_n)$, its empirical probability is

$$ p(x_1, x_2, \dots, x_n)$$

Then, $X$ is typical if

$$ 2^{-n(H(X) - \epsilon)} \leq  p(x_1, x_2, \dots, x_n) \leq 2^{-n(H(X) + \epsilon)}$$

where $H(X)$ is the entropy in bits. Back to your statement, maybe the 2nd law just means that excursions of a trajectory outside the typical set are exponentially unlikely as n goes to infinity. This is exactly what they call the asymptotic equipartition property

**Comment by ta-rekt** [comment]
I can do a simulation and post it here (TODO)

**Comment by ta-rekt** [comment]
![Screenshot from 2024-12-09 19-14-31](https://github.com/user-attachments/assets/e9fe03ee-e835-47f5-a0c4-ff9ac48bdbb2)
![Screenshot from 2024-12-09 19-14-51](https://github.com/user-attachments/assets/2b8351e4-cadc-4aed-a5a7-15604072dd66)
![Screenshot from 2024-12-09 19-15-06](https://github.com/user-attachments/assets/de4f7ee8-768d-409b-8627-d597553f17aa)

**Reply by ta-rekt** [reply]
from Elements of Information theory, by Cover and Thomas

**Reply by josePereiro** [reply]
This is great, questions:

> ...around the end of page 82: 
> A simple counterexample is provided by any Markov chain with a nonuniform stationary distribution. If we start this Markov chain from the uniform distribution, which is the maximum entropy distribution, the distribution will tend to the stationary distribution, which has a lower entropy than the uniform.

My question is related with equilibrium. Classical thermodynamics argue about entropy but referring always to equilibrium states, entropy is undefined otherwise. In statistical physics this is not required any more. Any time you have a distribution you can define an Entropy. So, the second law, which distribution (entropy) constraint? The distributions which evolve in the above analysis, do they describe a set of equilibrium? A kind of quasi-static process?

The second law states that in an isolated system IN EQUILIBRIUM, entropy never decreases. That means that if you force the system to change equilibrium state, but keeping it in isolation (no energy/matter exchange), the next final equilibrium state will have equal or larger entropy. Think about a box with a lot of compartments. The box is isolated. A gas is placed in one compartment.  By removing compartment walls you can move the system from one equilibrium state to others without breaking isolation. The entropy of the gas can no decrease. The 2nd law says that there is no procedure you can do, without breaking  isolation, to set the system into an new equilibrium state with lower entropy. The distributions describing the state of the gas in the dynamic phase between equilibrium states is not restricted by the 2nd law. The entropy of those distributions can do whatever.

**Reply by josePereiro** [reply]
Then, there are non equilibrium processes that appear to be better described by the Markov chain evolution. For instance, diffusion. What does the 2nd law says about diffusion? It is a dynamic process.
